wdn/templates_4.1/scripts/tabs.js

```diff
@@ -64,6 +66,70 @@ define(['jquery', 'wdn', 'require'], function($, WDN, require) {
 			nsel = trig.closest('li').siblings('.' + selected);
 			trig.trigger('tabchanged', [curr, nsel, tabs]);
 		},
+		
+		setupA11y = function ($tabsWithSwitch) {
+			$tabsWithSwitch.attr('role', 'tablist');
+
+			var $tabs = $($tabsWithSwitch.find('a'));
+
+			$tabs.each(function(index, element) {
+				console.log(element);
+				var $tab = $(element);
+
+				//Get the panel for this tab
+				var $panel = $('#'+getHashFromLink(element));
+
+				//If this tab doesn't have an id, give it one because we will need it later
+				if ('undefined' === typeof $tab.attr('id')) {
+					$tab.attr('id', 'wdn-tab-for-'+$panel.attr('id'));
+				}
+
+				//The parent <li> should not be read as a list item
+				$tab.parent('li').attr('role', 'presentation');
+
+				//Make sure that this <a> is read as a tab
+				$tab.attr('role', 'tab');
+
+				//Default to not selected
+				$tab.attr('aria-selected', 'false');
+
+				//Tell the tab that it controls this panel
+				$tab.attr('aria-controls', $panel.attr('id'));
+				
+				//Only make tabs pragmatically focusable by default
+				$tab.attr('tabindex', '-1');
+				
+				//Use arrow keys to focus tabs (standard tab interaction)
+				$tab.bind({
```

Comment:
Please don't use the bind method. I recommend adding this event listener to the `$tabsWithSwitch` and using delegation if necessary. See `.on()` API documentation.